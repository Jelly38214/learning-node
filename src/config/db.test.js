const { MYSQL_CONF, REDIS_CONF } = require("./db");

describe("Config For MYSQL & REDIS", () => {
  describe("MYSQL", () => {
    it("defaults", () => {
      // Arrange
      // Act
      // Assert
      expect(MYSQL_CONF).toMatchSnapshot();
    });
  });
  describe("REDIS", () => {
    it("defaults", () => {
      // Arrange
      // Act
      // Assert
      expect(REDIS_CONF).toMatchSnapshot();
    });
  });
});
