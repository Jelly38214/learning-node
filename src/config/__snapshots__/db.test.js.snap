// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Config For MYSQL & REDIS MYSQL defaults 1`] = `
Object {
  "database": "myblog",
  "host": "localhost",
  "password": "c11090201",
  "port": "3306",
  "user": "root",
}
`;

exports[`Config For MYSQL & REDIS REDIS defaults 1`] = `
Object {
  "host": "127.0.0.1",
  "port": 6379,
}
`;
